core/src/main/java/org/elasticsearch/common/settings/ClusterSettings.java

```diff
@@ -199,6 +199,7 @@ public void apply(Settings value, Settings current, Settings previous) {
                     ThrottlingAllocationDecider.CLUSTER_ROUTING_ALLOCATION_NODE_CONCURRENT_RECOVERIES_SETTING,
                     DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_LOW_DISK_WATERMARK_SETTING,
                     DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_HIGH_DISK_WATERMARK_SETTING,
+                    DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_FLOOD_STAGE_SETTING,
```

Comment:
can we add DISK to this name (with or without watermark - see other comment). i.e. `CLUSTER_ROUTING_ALLOCATION_FLOOD_STAGE_DISK_WATERMARK` or something?